examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,858 @@
+       "      [1., 1., 1., 2., 1., 1., 1., 1., 1., 1.],\n",
+       "      [1., 1., 1., 1., 2., 1., 1., 1., 1., 1.],\n",
+       "      [1., 1., 1., 1., 1., 2., 1., 1., 1., 1.],\n",
+       "      [1., 1., 1., 1., 1., 1., 2., 1., 1., 1.],\n",
+       "      [1., 1., 1., 1., 1., 1., 1., 2., 1., 1.],\n",
+       "      [1., 1., 1., 1., 1., 1., 1., 1., 2., 1.],\n",
+       "      [1., 1., 1., 1., 1., 1., 1., 1., 1., 2.]], dtype=float32)"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from devito import Eq, Operator\n",
+    "op = Operator(Eq(f, f + 1))\n",
+    "op.apply()\n",
+    "f.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Every value has been updated by one. You should be able to see twos on the diagonal and ones everywhere else."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "#define _POSIX_C_SOURCE 200809L\n",
+      "#include \"stdlib.h\"\n",
+      "#include \"math.h\"\n",
+      "#include \"sys/time.h\"\n",
+      "#include \"xmmintrin.h\"\n",
+      "#include \"pmmintrin.h\"\n",
+      "\n",
+      "struct dataobj\n",
+      "{\n",
+      "  void *restrict data;\n",
+      "  int * size;\n",
+      "  int * npsize;\n",
+      "  int * dsize;\n",
+      "  int * hsize;\n",
+      "  int * hofs;\n",
+      "  int * oofs;\n",
+      "} ;\n",
+      "\n",
+      "struct profiler\n",
+      "{\n",
+      "  double section0;\n",
+      "} ;\n",
+      "\n",
+      "\n",
+      "int Kernel(struct dataobj *restrict f_vec, struct profiler * timers, const int x_M, const int x_m, const int y_M, const int y_m)\n",
+      "{\n",
+      "  float (*restrict f)[f_vec->size[1]] __attribute__ ((aligned (64))) = (float (*)[f_vec->size[1]]) f_vec->data;\n",
+      "  /* Flush denormal numbers to zero in hardware */\n",
+      "  _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);\n",
+      "  _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);\n",
+      "  struct timeval start_section0, end_section0;\n",
+      "  gettimeofday(&start_section0, NULL);\n",
+      "  /* Begin section0 */\n",
+      "  for (int x = x_m; x <= x_M; x += 1)\n",
+      "  {\n",
+      "    #pragma omp simd aligned(f:32)\n",
+      "    for (int y = y_m; y <= y_M; y += 1)\n",
+      "    {\n",
+      "      f[x + 1][y + 1] = f[x + 1][y + 1] + 1;\n",
+      "    }\n",
+      "  }\n",
+      "  /* End section0 */\n",
+      "  gettimeofday(&end_section0, NULL);\n",
+      "  timers->section0 += (double)(end_section0.tv_sec-start_section0.tv_sec)+(double)(end_section0.tv_usec-start_section0.tv_usec)/1000000;\n",
+      "  return 0;\n",
+      "}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "print(op.ccode) # Print the generated code"
```

Comment:
I would comment out this line. This is the 5th (!) tutorial in the series, and by this point, users will know what an Operator like that does

Reply:
also, you should remove the NBVAL here

Reply:
Fixed